librz/core/cmd_debug.c

```diff
@@ -5036,6 +4410,655 @@ RZ_IPI int rz_cmd_debug(void *data, const char *input) {
+	if (argc > 1) {
+		RzBreakpointItem *bpi;
+		int index = rz_num_math(core->num, argv[1]);
+		if ((bpi = rz_bp_get_index(core->dbg->bp, index))) {
+			bpi->enabled = true;
+		} else {
+			RZ_LOG_ERROR("Unable to find breakpoint with index %d\n", index);
+		}
+	} else {
+		return RZ_CMD_STATUS_WRONG_ARGS;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+// dbid
+RZ_IPI RzCmdStatus rz_cmd_debug_disable_bp_index_handler(RzCore *core, int argc, const char **argv) {
+	if (argc > 1) {
+		RzBreakpointItem *bpi;
+		int index = rz_num_math(core->num, argv[1]);
+		if ((bpi = rz_bp_get_index(core->dbg->bp, index))) {
+			bpi->enabled = false;
+		} else {
+			RZ_LOG_ERROR("Unable to find breakpoint with index %d\n", index);
+		}
+	} else {
+		return RZ_CMD_STATUS_WRONG_ARGS;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+// dbis
+RZ_IPI RzCmdStatus rz_cmd_debug_toggle_bp_index_handler(RzCore *core, int argc, const char **argv) {
+	if (argc > 1) {
+		RzBreakpointItem *bpi;
+		int index = rz_num_math(core->num, argv[1]);
+		if ((bpi = rz_bp_get_index(core->dbg->bp, index))) {
+			bpi->enabled = !bpi->enabled;
+		} else {
+			RZ_LOG_ERROR("Unable to find breakpoint with index %d\n", index);
+		}
+	} else {
+		return RZ_CMD_STATUS_WRONG_ARGS;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+// dbite
+RZ_IPI RzCmdStatus rz_cmd_debug_enable_bp_trace_index_handler(RzCore *core, int argc, const char **argv) {
+	if (argc > 1) {
+		RzBreakpointItem *bpi;
+		int index = rz_num_math(core->num, argv[1]);
+		if ((bpi = rz_bp_get_index(core->dbg->bp, index))) {
+			bpi->trace = true;
+		} else {
+			RZ_LOG_ERROR("Unable to find breakpoint with index %d\n", index);
+		}
+	} else {
+		return RZ_CMD_STATUS_WRONG_ARGS;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+// dbitd
+RZ_IPI RzCmdStatus rz_cmd_debug_disable_bp_trace_index_handler(RzCore *core, int argc, const char **argv) {
+	if (argc > 1) {
+		RzBreakpointItem *bpi;
+		int index = rz_num_math(core->num, argv[1]);
+		if ((bpi = rz_bp_get_index(core->dbg->bp, index))) {
+			bpi->trace = false;
+		} else {
+			RZ_LOG_ERROR("Unable to find breakpoint with index %d\n", index);
+		}
+	} else {
+		return RZ_CMD_STATUS_WRONG_ARGS;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+// dbits
+RZ_IPI RzCmdStatus rz_cmd_debug_toggle_bp_trace_index_handler(RzCore *core, int argc, const char **argv) {
+	if (argc > 1) {
+		RzBreakpointItem *bpi;
+		int index = rz_num_math(core->num, argv[1]);
+		if ((bpi = rz_bp_get_index(core->dbg->bp, index))) {
+			bpi->trace = !bpi->enabled;
+		} else {
+			RZ_LOG_ERROR("Unable to find breakpoint with index %d\n", index);
+		}
+	} else {
+		return RZ_CMD_STATUS_WRONG_ARGS;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+// dbh
+RZ_IPI RzCmdStatus rz_cmd_debug_bp_plugin_handler(RzCore *core, int argc, const char **argv) {
+	if (argc == 1) {
+		rz_bp_plugin_list(core->dbg->bp);
+	} else if (argc == 2) {
+		if (!rz_bp_use(core->dbg->bp, argv[1], core->analysis->bits)) {
+			RZ_LOG_ERROR("Failed to set breakpoint plugin handler to %s\n", argv[1]);
+		}
+	} else {
```

Comment:
No need for these `RZ_CMD_STATUS_WRONG_ARGS` everywhere to check *just* the count of the arguments.